tests/Computation.Tests/ConstantDoubleTests.cs

```diff
@@ -0,0 +1,437 @@
+            SymbolicFloat symbolicLeft, SymbolicFloat symbolicRight)
+        {
+            var constant = constantLeft.FloatSubtract(constantRight).GetDoubleNanNormalizedInteger(Context);
+            var symbolic = symbolicLeft.FloatSubtract(symbolicRight).GetDoubleNanNormalizedInteger(Context);
+
+            constant.Should().Be(symbolic);
+        }
+
+        [Theory]
+        [ClassData(typeof(FloatToSignedTestData))]
+        private void ShouldCreateEquivalentConstantAndSymbolicValuesForFloatToSigned(Bits size,
+            ConstantDouble constantExpression,
+            SymbolicFloat symbolicExpression)
+        {
+            var constant = constantExpression.FloatToSigned(size).GetInteger(Context);
+            var symbolic = symbolicExpression.FloatToSigned(size).GetInteger(Context);
+
+            constant.Should().Be(symbolic);
+        }
+
+        [Theory]
+        [ClassData(typeof(FloatToUnsignedTestData))]
+        private void ShouldCreateEquivalentConstantAndSymbolicValuesForFloatToUnsigned(Bits size,
+            ConstantDouble constantExpression,
+            SymbolicFloat symbolicExpression)
+        {
+            var constant = constantExpression.FloatToUnsigned(size).GetInteger(Context);
+            var symbolic = symbolicExpression.FloatToUnsigned(size).GetInteger(Context);
+
+            constant.Should().Be(symbolic);
+        }
+
+        [Theory]
+        [ClassData(typeof(BinaryTestData))]
+        private void ShouldCreateEquivalentConstantAndSymbolicValuesForFloatUnordered(
+            ConstantDouble constantLeft, ConstantDouble constantRight,
+            SymbolicFloat symbolicLeft, SymbolicFloat symbolicRight)
+        {
+            var constant = constantLeft.FloatUnordered(constantRight).GetInteger(Context);
+            var symbolic = symbolicLeft.FloatUnordered(symbolicRight).GetInteger(Context);
+
+            constant.Should().Be(symbolic);
+        }
+
+        private sealed class UnaryTestData : TheoryData<
+            ConstantDouble,
+            SymbolicFloat>
+        {
+            public UnaryTestData()
+            {
+                foreach (var value in Values())
+                    Add(new ConstantDouble(value),
+                        new SymbolicFloat((Bits) 64U, c => c.MkFP(value, c.MkFPSort64())));
+            }
+
+            private static IEnumerable<double> Values()
+            {
+                yield return 0d;
+                yield return double.Epsilon;
+                yield return double.NaN;
+                yield return double.NegativeInfinity;
+                yield return double.MinValue;
+                yield return double.MaxValue;
+                yield return double.PositiveInfinity;
+
+                foreach (var i in Enumerable.Range(-10, 10))
+                {
+                    yield return i / 3d;
+                    yield return 3d / i;
+                }
+
+                foreach (var i in Enumerable.Range(1, 10))
```

Comment:
Isn't this range already covered above?

Reply:
Of course. I was being stupid and read the first one as the range `[-10..10]` as you've probably guessed.